elk/utils/data_utils.py

```diff
@@ -49,13 +48,16 @@ def get_columns_all_equal(dataset: DatasetDict) -> list[str]:
     return pivot
 
 
-def select_train_val_splits(raw_splits: Iterable[str]) -> tuple[str, str]:
-    """Return splits to use for train and validation, given an Iterable of splits."""
-    priorities = {
+def select_train_val_splits(
+    raw_splits: Iterable[str],
+    priorities: dict = {
```

Comment:
I don't think the priorities should be customizable here. This also violates the usual rule that you shouldn't have _mutable_ objects as default values for function arguments, because really weird stuff can happen